### DISCUSSION TITLE
per issue fork, closeable fork, light fork, issues sample repository

### DISCUSSION THREAD

**Discussion by xenoterracide** [community/community#108288] [Repositories]
### Select Topic Area

Product Feedback

### Body

So, I have/had something like 150 repositories, some of these existed for either me fixing upstream bugs, or for giving a sample for a bug. Gist's are often insufficient because they have no file hierarchy (although maybe they could be improved). My suggestion I believe crosses many of the various categories. I'd like to be able to do the following steps as a workflow

1. Open Upstream issues
1. create or fork for issue
2. repository is created with a name based on the `<user/org>/<repo>/<issue>#subid` that was created.
   * the repository may be created with the contents of a skeleton provided by the `<user/org>/<repo>`
   * subid exists so we can have more than one repo created for an issue.
1. This repository does not show up in my repository list, or is filtered by default.
1. When the issue is closed the repository is further hidden or even deleted.

Basically, I don't want to have to track/worry about repositories that are for reporting/fixing 3rd party issues where I'm not constantly contributing.
